sdcm/docker.py

```diff
@@ -215,33 +239,150 @@ def destroy(self):
             node.destroy(force=True)
 
 
-class ScyllaDockerCluster(DockerCluster, cluster.BaseScyllaCluster):  # pylint: disable=abstract-method
+class ScyllaDockerCluster(cluster.BaseScyllaCluster, DockerCluster):  # pylint: disable=abstract-method
 
     def __init__(self, **kwargs):
-        self._user_prefix = kwargs.get('user_prefix', cluster.DEFAULT_USER_PREFIX)
-        self._node_prefix = '%s-%s' % (self._user_prefix, BASE_NAME)
-        super(ScyllaDockerCluster, self).__init__(node_prefix=kwargs.get('node_prefix', self._node_prefix),
+        user_prefix = kwargs.get('user_prefix')
+        cluster_prefix = cluster.prepend_user_prefix(user_prefix, 'db-cluster')
+        node_prefix = cluster.prepend_user_prefix(user_prefix, 'db-node')
+
+        self.aws_extra_network_interface = False  # TODO: move to Base
+        super(ScyllaDockerCluster, self).__init__(node_prefix=node_prefix,
+                                                  cluster_prefix=cluster_prefix,
                                                   **kwargs)
 
-    @retrying(n=30, sleep_time=3, allowed_exceptions=(cluster.ClusterNodesNotReady, DockerCommandError))
+    def node_setup(self, node, verbose=False, timeout=3600):
+        endpoint_snitch = self.params.get('endpoint_snitch')
+        seed_address = ','.join(self.seed_nodes_ips)
+
+        node.wait_ssh_up(verbose=verbose)
+        # node.install_scylla_debuginfo()
```

Comment:
think if we want this every time, take a while to download and install

Reply:
also need to make sure it takes the correct ones

Reply:
maybe we should put a configuration parameter to enable disable the install of debug info ? so the default would be speedy

Reply:
it's now will be under `if Setup.BACKTRACE_DECODING:`